nexus/db-queries/src/db/lookup.rs

```diff
@@ -540,6 +579,15 @@ lookup_resource! {
 
 lookup_resource! {
     name = "Image",
+    ancestors = ["Silo"],
+    children = [],
+    lookup_by_name = false,
```

Comment:
It's notable that this resource is _only_ used for ID lookups. If there's a name it's always either `SiloImage` or `ProjectImage` instead.